client/my-sites/domains/domain-management/transfer/transfer-out/unlocked.jsx

```diff
@@ -11,27 +11,53 @@ import Card from 'components/card';
 import SectionHeader from 'components/section-header';
 import { getSelectedDomain } from 'lib/domains';
 import Button from 'components/button';
-import { requestTransferCode, enableDomainLocking } from 'lib/upgrades/actions';
+import { requestTransferCode, cancelTransferRequest } from 'lib/upgrades/actions';
 import notices from 'notices';
 import { displayRequestTransferCodeResponseNotice } from './shared';
 import support from 'lib/url/support';
 
 class Unlocked extends React.Component {
-	state = {
-		submitting: false
-	};
+
+	constructor( props ) {
+		super( props );
+
+		const {
+			pendingTransfer,
+			domainLockingAvailable,
+			hasPrivacyProtection,
+		} = getSelectedDomain( this.props );
+
+		this.state = { submitting: false, canceled: false };
+
+		if ( ! ( pendingTransfer || domainLockingAvailable || hasPrivacyProtection ) ) {
```

Comment:
Unless I'm missing something, I don't think sending the request code right away when landing on the `Transfer Out` page makes sense.